openthread/Kconfig

```diff
@@ -79,32 +89,14 @@ config OPENTHREAD_NORDIC_LIBRARY_MASTER
 
 config OPENTHREAD_NORDIC_LIBRARY_FTD
 	bool "Nordic Optimized FTD configuration"
-	select OPENTHREAD_COAP
-	select OPENTHREAD_DNS_CLIENT
-	select OPENTHREAD_ECDSA
-	select OPENTHREAD_EXTERNAL_HEAP
-	select OPENTHREAD_IP6_FRAGM
-	select OPENTHREAD_NETDIAG_CLIENT
-	select OPENTHREAD_SLAAC
-	select OPENTHREAD_SRP_CLIENT
-	select OPENTHREAD_DUA if !OPENTHREAD_THREAD_VERSION_1_1
-	select OPENTHREAD_MLR if !OPENTHREAD_THREAD_VERSION_1_1
+	select OPENTHREAD_NORDIC_LIBRARY_COMMON
 	select OPENTHREAD_LINK_METRICS_SUBJECT if !OPENTHREAD_THREAD_VERSION_1_1
 	help
 	  Nordic Semiconductor optimized OpenThread features for FTD.
 
 config OPENTHREAD_NORDIC_LIBRARY_MTD
 	bool "Nordic Optimized MTD configuration"
-	select OPENTHREAD_COAP
-	select OPENTHREAD_DNS_CLIENT
-	select OPENTHREAD_ECDSA
-	select OPENTHREAD_EXTERNAL_HEAP
-	select OPENTHREAD_IP6_FRAGM
-	select OPENTHREAD_NETDIAG_CLIENT
-	select OPENTHREAD_SLAAC
-	select OPENTHREAD_SRP_CLIENT
-	select OPENTHREAD_DUA if !OPENTHREAD_THREAD_VERSION_1_1
-	select OPENTHREAD_MLR if !OPENTHREAD_THREAD_VERSION_1_1
+	select OPENTHREAD_NORDIC_LIBRARY_COMMON
```

Comment:
We don't want to enable TCP in the FTD and MTD libraries. As far as i understand TCP is required by component, but it's not required for end-products. Thus i would keep it enabled only for Master variant.

Reply:
right! fixed